add_clip_ui.ui

```diff
@@ -43,11 +43,14 @@
    </property>
   </widget>
   <widget class="QRadioButton" name="newButton">
+   <property name="enabled">
+    <bool>true</bool>
```

Comment:
I'm actually not editing the .ui files manually, but with Qt Designer, so it must have put the tag there.